scripts/process_jartic_parallel.py

```
@@ -0,0 +1,320 @@
+
+
+def process_prefecture_data(pref_data_tuple):
+    import gc
+    import tempfile
+    pref_zip_name, pref_bytes = pref_data_tuple
+    
+    try:
+        parts = pref_zip_name.split('_')
+        prefecture = parts[1] if len(parts) >= 2 else "unknown"
+        
+        temp_file = tempfile.NamedTemporaryFile(mode='w', delete=False, suffix='.csv', encoding='utf-8')
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)